index.js

```diff
@@ -262,6 +262,12 @@ module.exports = fp(
                 ? staticMode.bindSetMetric(stats)
                 : dynamicMode.bindSetMetric(stats)
         );
+        fastify.decorateRequest(
+            'getLabel',
+            mode === 'static'
+                ? staticMode.bindGetLabel(stats)
```

Comment:
There is no need here to bind the client, we can pass directly the function that resolves the label 😉

Reply:
> There's a small drop in the coverage percentage, are we missing some important code paths?

Missing some test, It is not completed yet.

Reply:
> There is no need here to bind the client, we can pass directly the function that resolves the label wink

In static mode no, in dynamic yes. or I missing something?

Reply:
The bind is to associate a stats client to to a function that sends metrics. In this case we don't need the client because we are just returning the label string, in both modes, that is why the bind is not necessary.

Reply:
> Missing some test, It is not completed yet.

Sorry, I just saw the is a draft.